### ISSUE TITLE
False Positive: MethodNames

### ISSUE THREAD

**Issue by hzwaal** [sksamuel/scapegoat#148]
When trying to define a custom assignment operator, for instance:

```scala
class Thing(private var v: Int) {
  def value: Int = v
  def value_=(v: Int): Unit = {
    require(v >= 0)
    this.v = v
  }
}
```

I get the following message:

```
[info] scapegoat.scala:4: Method name not recommended
          Methods should be in camelCase style with the first letter lower-case. See http://docs.scala-lang.org/style/naming-conventions.html#methods

[info] Thing.scala:4: Method name not recommended
[info]     def value_=(v: Int): Unit = {
[info]         ^
```

Which I think is a false positive, because this is the correct way to implement an assignment operator.

**Comment by fkz** [comment]
this is fixed in the newest version 1.3.0

**Comment by hzwaal4c** [comment]
Yes, I've verified this and you're right.

This issue can be closed.
